packer/pmm2.json

```diff
@@ -1,4 +1,9 @@
 {
+    "variables": {
+      "single_disk": "false",
+      "pmm2_server_repo": "testing",
+      "pmm_client_repos": "original testing"
+    },
```

Comment:
It seems we also need `pmm_client_repo_name` here:
https://github.com/percona/pmm-server/blob/PMM-2.0/ansible/roles/pmm2-images/tasks/main.yml#L174
`"pmm_client_repo_name": "percona-testing-x86_64"`